molotov/listeners.py

```diff
@@ -0,0 +1,88 @@
+import io
+
+
+_UNREADABLE = "***WARNING: Molotov can't display this body***"
+_BINARY = "**** Binary content ****"
+_FILE = "**** File content ****"
+_COMPRESSED = ('gzip', 'compress', 'deflate', 'identity', 'br')
+
+
+class BaseListener(object):
+    async def __call__(self, session, event, **options):
+        attr = getattr(self, 'on_' + event)
+        await attr(**options)
```

Comment:
Could this instead be something like 
```python
attr = getattr(self, 'on_' + event, None)
if attr is not None:
    await attr(**options)
```
That way, if I writer a listener class (based on `BaseListener`) and I'm *only* interested in `on_response_received` I wouldn't *have to* write...
```python
    async def on_receive_request(self, *i, **dontcare):
            pass
```

Reply:
Good idea